bin/run-clara

```diff
@@ -73,7 +74,9 @@ done
 # Set some JVM options:
 export JAVA_OPTS="$JAVA_OPTS -XX:+IgnoreUnrecognizedVMOptions"
 export JAVA_OPTS="$JAVA_OPTS -Djava.io.tmpdir=$CLARA_USER_DATA -Dorg.sqlite.tmpdir=$CLARA_USER_DATA"
-export JAVA_OPTS="$JAVA_OPTS -Djava.util.logging.config.file=$CLAS12DIR/etc/logging/debug.properties"
+# Set verbosity:
+[ -z ${quiet+x} ] && stub=debug || stub=info
+export JAVA_OPTS="$JAVA_OPTS -Djava.util.logging.config.file=$CLAS12DIR/etc/logging/$stub.properties"
```

Comment:
`debug.properties` no longer exists; either add that file or change the possible values of `$stub`.